frontend/src/pages/RegionalDashboard/components/RecipientSpotlightDashboard.scss

```diff
@@ -0,0 +1,17 @@
+/* Custom styling for Recipient Spotlight Dashboard overview widget */
+
+.recipient-spotlight-overview {
+  max-width: 337px;
```

Comment:
This is kind of arbitrary, we could probably use the grid or one of the USWDS max widths instead of introducing "337px" into our code